packages/aragon-wrapper/src/rpc/handlers/external.js

```diff
@@ -16,6 +16,35 @@ export function call (request, proxy, wrapper) {
   return contract.methods[method.name](...params).call()
 }
 
+export async function externalIntent (request, proxy, wrapper) {
+  const [
+    proxyAddress,
+    method,
+    ...params
+  ] = request.params
+
+  const installedApp = await wrapper.getApp(proxyAddress)
+  const transactionPath = installedApp
+    ? await wrapper.getTransactionPath(
+      proxyAddress,
+      method.name,
+      params
+    )
+    : await wrapper.getUninstalledAppTransactionPath(
+      proxyAddress,
+      method,
+      params
+    )
+
+  return wrapper.performTransactionPath(
+    transactionPath.map((tx) => ({
+      ...tx,
+      external: true,
+      installedApp: !!installedApp
```

Comment:
As mentioned in https://github.com/aragon/aragon/pull/850#discussion_r299933995, we don't _really_ need this, since we can check in the client if the `to` is an installed app or not.

Reply:
Are you talking just about the `installedApp` var or `external` too? It seems apps would need a way to determine their own proxy address to determine `external`, but `installedApp` could definitely be calculated easily.

Reply:
Just `installedApp`. `external` is a little bit harder to get because it requires some context about the currently running app (which the client does still have, but involves a bit more state plumbing).